OWML.ModLoader/ModData.cs

```diff
@@ -11,134 +11,34 @@ public class ModData : IModData
     {
         public IModManifest Manifest { get; }
         public IModConfig Config { get; private set; }
+        [Obsolete("Maybe this should be obsolete?")]
         public IModConfig DefaultConfig { get; private set; }
```

Comment:
As far as I can tell this isn't used anywhere else in OWML, so it could be removed. Could mark as obsolete, but there isn't really an alternative.

Reply:
try removing it, if it causes issues we'll know when testing ;)